pkg/utils/kubernetes/kubernetes.go

```diff
@@ -248,6 +248,14 @@ func ObjectFromTypedObjectReference(ref *v1alpha1.TypedObjectReference) *unstruc
 	}
 }
 
+// ObjectKeyFromTypedObjectReference returns the object key (NamespacedName) from a TypedObjectReference
+func ObjectKeyFromTypedObjectReference(obj *v1alpha1.TypedObjectReference) *types.NamespacedName {
```

Comment:
isn't this function already available on the v1alpha1.ObjectReference?

Reply:
I am not even using it anywhere... Removed.